### PR TITLE
Add missing parameters/properties docs

### PR DESCRIPTION
**Pull request by buyaa-n**
## Summary

Fix undocumented 7.0 APIs in System.Runtime.Versioning

Fixes https://github.com/dotnet/runtime/issues/74733

### CODE REVIEW

File: xml/System.Runtime.Versioning/ObsoletedInOSPlatformAttribute.xml [hunk 1]
```diff
@@ -70,9 +70,9 @@ This attribute is primarily used by OS bindings to indicate APIs that shouldn't
         <Parameter Name="message" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="platformName">To be added.</param>
-        <param name="message">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="platformName">The obsoleted OS platform name, optionally including a version.</param>
+        <param name="message">Additional message that suggests a better alternative for the obsoleted API.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.Runtime.Versioning.ObsoletedInOSPlatformAttribute" /> attribute class for the specified obsoleted OS platform with an additional message</summary>
```
Comment (carlossanlop) [inline xml/System.Runtime.Versioning/ObsoletedInOSPlatformAttribute.xml:75]:
```suggestion
        <summary>Initializes a new instance of the <see cref="T:System.Runtime.Versioning.ObsoletedInOSPlatformAttribute" /> attribute class for the specified obsoleted OS platform with an additional message.</summary>
```

File: xml/System.Runtime.Versioning/ObsoletedInOSPlatformAttribute.xml [hunk 2]
```diff
@@ -92,8 +92,8 @@ This attribute is primarily used by OS bindings to indicate APIs that shouldn't
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the additional message provided for the attribute, or null.</summary>
+        <value>Provides additional message about the obsoleted API, mostly suggests a better alternative for the OS platform specified.</value>
         <remarks>To be added.</remarks>
```
Comment (carlossanlop) [inline xml/System.Runtime.Versioning/ObsoletedInOSPlatformAttribute.xml:97]: What do you think of this rewording? I also moved the value to the remarks.
```suggestion
        <summary>Gets the optional additional message provided for the attribute.</summary>
        <value>A string that represents the message, or <see langword="null"/> if there is none.</value>
        <remarks>Provides an additional message about the obsoleted API, mostly to suggest a better alternative for the OS platform specified.</remarks>
```

File: xml/System.Runtime.Versioning/UnsupportedOSPlatformAttribute.xml [hunk 3]
```diff
@@ -100,8 +100,8 @@ The versionless attribute is primarily used to indicate the API is unsupported f
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the additional message provided for the attribute, or null.</summary>
+        <value>Provides additional message about the unsupported API, mostly suggest a replacement API for the OS platform specified.</value>
         <remarks>To be added.</remarks>
```
Comment (carlossanlop) [inline xml/System.Runtime.Versioning/UnsupportedOSPlatformAttribute.xml:105]: Same, what do you think?
```suggestion
        <summary>Gets the optional additional message provided for the attribute.</summary>
        <value>A string that represents the message, or <see langword="null"/> if there is none.</value>
        <remarks>Provides an additional message about the unsupported API, mostly to suggest a replacement API for the OS platform specified.</remarks>
```

File: xml/System.Runtime.Versioning/ObsoletedInOSPlatformAttribute.xml (hunk 2 above)
Comment (buyaa-n) [reply xml/System.Runtime.Versioning/ObsoletedInOSPlatformAttribute.xml:97]: Looks good, thanks

**Pull request merged**